src/routes/index.svelte

```diff
@@ -2,4 +2,168 @@
 	import { base } from '$app/paths';
 </script>
 
-<a href="{base}/hot">Hot vacancies</a>
+<svelte:head>
+	<title>JobFinder - Homepage</title>
+</svelte:head>
+
+<div class="main-block">
+	<img src="{base}/logo_main.png" alt="JobFinder logo" />
+	<a href="{base}/filter"><button class="btn buton">Discover Jobs</button></a>
```

Comment:
`.buton` selector is not used anywhere, except for this button, so `.discovery-button` would be a more descriptive name.